.github/workflows/trivy-image-scan.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           image-ref: ${{ inputs.IMAGE_REF }}
       - name: Release SBOM for image (IF has tags for release)
         if: startsWith(github.ref, 'refs/tags/')
-        uses: softprops/action-gh-release@v4
+        uses: softprops/action-gh-release@v2
```

Comment:
Why role back to such an old version of the action?

Reply:
"v4" was copy-pasta error. It doesn't exist  - v2 is the latest version